src/js/events/CosmosPackagesActions.js

```diff
@@ -165,14 +165,15 @@ const CosmosPackagesActions = {
 
         AppDispatcher.handleServerAction({
           type: REQUEST_COSMOS_PACKAGE_LIST_VERSIONS_SUCCESS,
-          data: packageVersions
+          data: packageVersions,
+          packageName
         });
       },
       error(xhr) {
         AppDispatcher.handleServerAction({
           type: REQUEST_COSMOS_PACKAGE_LIST_VERSIONS_ERROR,
           data: RequestUtil.getErrorFromXHR(xhr),
-          xhr
```

Comment:
Are we sure that we don't need the xhr anymore? @MatApple didn't you add the native xhr to almost all actions to handle permission issues?

Reply:
We're not actually using the xhr object. The permissions error handling was never completed and I opposed the way we attempted to complete it.

Reply:
Added it here for consistency.